addon/const/osf-services.js

```diff
@@ -0,0 +1,38 @@
+import serviceLinks from './service-links';
+
+/**
+ * @module ember-osf
+ * @submodule const
+ */
+
+/**
+ * @class osf-services
+ */
+
+/**
+ * Provides list of OSF services and their links
+ * @property osfServices
+ * @final
+ * @type {Array}
+ */
+
+const osfServices = [
+    {
+        name: 'HOME',
+        url: serviceLinks.osfHome,
+    },
+    {
+        name: 'PREPRINTS',
+        url: serviceLinks.preprintsHome
+    },
+    {
+        name: 'REGISTRIES',
+        url: serviceLinks.registriesHome
+    },
+    {
+        name: 'MEETINGS',
+        url: serviceLinks.meetingsHome
+    }
+];
+
+export default osfServices;
```

Comment:
I don't think they should be combined. osf-services is mainly going to be used on this new navbar.  However, the service links can be used everywhere there is a link.  osf-services needs service-links, but not the other way around.